### DISCUSSION TITLE
include-templates issue in config files

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei#2738] [Q&A]
Hey, I encountered an issue with making config file.
First I excluded the info severity templates to reduce the number of request, but I also want to check 300 info-severity templates that are interesting for me. So I used the include-templates in the config file, but what I realized is that it added approximately 8000 new requests, which is very weird (all the templates together are between 5000-6000 requests if im not wrong).

This is what I get with the -stats flag:
[INF] Templates loaded for scan: 2491
[INF] Templates clustered: 95 (Reduced 81 HTTP Requests)
[0:00:05] | Templates: 2491 | Hosts: 1 | RPS: 3 | Matched: 0 | Errors: 1 | Requests: 17/10485 (0%)
[INF] Using Interactsh Server: oast.live
[0:00:10] | Templates: 2491 | Hosts: 1 | RPS: 3 | Matched: 0 | Errors: 3 | Requests: 39/10485 (0%)

**Comment by ehsandeep** [comment]
@Y0b4g3t please do share cmd / config you used for the same.

**Comment by ghost** [comment]
https://github.com/Y0b4g3t/thing/blob/main/nuclei2.yaml

CMD:
nuclei -u "https://redacted" -config nuclei2.yaml -stats
